NorthstarDedicatedTest/dllmain.cpp

```diff
@@ -65,6 +72,48 @@ void WaitForDebugger(HMODULE baseAddress)
 	}
 }
 
+bool LoadPlugins() {
+
+	std::vector<fs::path> paths;
+
+	// ensure dirs exist
+	fs::create_directories("plugins");
+
+	// get mod directories
+
+	for (auto const& entry : fs::recursive_directory_iterator("plugins"))
+	{
+		if (fs::is_regular_file(entry) && entry.path().extension() == ".dll")
+			paths.emplace_back(entry.path().filename());
+	}
+
+	initGameState();
+	spdlog::info("Loading the following DLLs in plugins folder:");
+	for (fs::path path : paths)
+	{
+		std::string pathstring = ("plugins"/ path).string();
+		std::wstring wpath = ("plugins"/path).wstring();
+		
+		LPCWSTR wpptr = wpath.c_str();
+		//spdlog::info(wpptr);
+		HMODULE hLib = LoadLibraryW(wpptr);
```

Comment:
You will need to resolve the absolute path or plugins with the same basename as a loaded library or a lib on the PATH won't work correctly. Also, use `LOAD_WITH_ALTERED_SEARCH_PATH`.

Reply:
https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-getfullpathnamew